modules/serverless-autoscaling-maxscale-kn.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="serverless-autoscaling-maxscale-kn_{context}"]
-= Setting the maxScale annotation by using the Knative CLI
+= Setting the max-scale annotation by using the Knative CLI
 
-Using the `kn` CLI to set the `maxScale` annotation provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn service` command with the `--max-scale` flag to create or modify the `--max-scale` value for a service.
+Using the `kn` CLI to set the `max-scale` annotation provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn service` command with the `--max-scale` flag to create or modify the `--max-scale` value for a service.
```

Comment:
I found another issue :sweat_smile: 
`kn service` takes `--scale-max` to set `max-scale` annotation. It is very confusing. I opened https://issues.redhat.com/browse/SRVCLI-349 (This may not be a bug, though) and https://issues.redhat.com/browse/SRVCLI-350 (This must be updated)

Reply:
@abrennan89 as Kenjiro pointed out we changed those flags in client. Would you mind creeping the change on top of this PR? 

```suggestion
Using the `kn` CLI to set the `max-scale` annotation provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn service` command with the `--scale-max` flag to create or modify the `--scale-max` value for a service.
```

Reply:
@dsimansk is there an engineering Jira for that change? This change should be mentioned in release notes for whatever version it was changed in, so we should at least have it tracked somewhere if we make the change in this PR.